guides/keras_nlp/getting_started.py

```diff
@@ -247,6 +254,13 @@
     sequence_length=512,
 )
 
+# Now apply the preprocessor to every sample of train and test data using `map()`.
```

Comment:
Maybe make this shorter and link our to a guide for those who want to go deeper.

```
# Apply the preprocessor to every sample of train and test data using `map()`.
# `tf.data.AUTOTUNE` and `prefetch()` are options to tune performance, see
# https://www.tensorflow.org/guide/data_performance
```

Reply:
I will update you with the changes soon.